pyproject.toml

```diff
@@ -4,7 +4,7 @@ build-backend = "flit_core.buildapi"
 
 [project]
 name = "ansys-templates"
-version = "0.5.dev0"
+version = "0.6.dev0"
```

Comment:
Please, remove this changes. No need to update the version in `main`. This is done in the `release/` branches.